### ISSUE TITLE
Cannot play sound using internal player after upgrade to Ubuntu 13.10.

### ISSUE THREAD

**Issue by metalbrick** [goldendict/goldendict#412]
The prompt information is:

"WARN: FFmpeg audio Player ao_open_live() failed: can not  open the device alsa channel 1, rate 11025, bit 8."

**Comment by loge** [comment]
I found a soulotion for the problem.
Install VLC and then in goldendict menu:
From Edit > Preferences > Audio > Use external program. And then add this line to the field : /usr/bin/cvlc
All done.

**Comment by Kasramvd** [comment]
@loge  It repeated the sound without stopping. Do you have such problem too?

**Comment by loge** [comment]
@Kasramvd No. Everything is good here. Except sometimes I have to run killall vlc because it stops to function.

**Comment by trienthuyendang** [comment]
> @loge It repeated the sound without stopping. Do you have such problem too?

If it repeats the sound, you can fix it by turning off 'loop all' or 'loop one' in VLC gui control bar, then exit VLC, then restart goldendict.

> @kasramvd No. Everything is good here. Except sometimes I have to run killall vlc because it stops to function.

You may want to try this: Open VLC Player on your PC > 'Media' on the tool bar at the top of VLC media player > check 'Quit at the end of playlist' > exit VLC > restart goldendict.

**Comment by xapaxuc** [comment]
> I found a soulotion for the problem.
> Install VLC and then in goldendict menu:
> From Edit > Preferences > Audio > Use external program. And then add this line to the field : /usr/bin/cvlc
> All done.

The problem is that the vlc-processes do not stop.
Use "/usr/bin/cvlc --play-and-exit" instead. It's works.

**Comment by Lian-LF** [comment]
On my manjaro, install the mpv, then just put the `mpv` as the external program work fine, no looping issue.

**Comment by Konfekt** [comment]
If there are many dictionaries loaded by GoldenDict, then there could also be too many files open when the temporary wave file to be opened. Reducing the number of loaded dictionaries or [increasing the process limit](https://unix.stackexchange.com/questions/8945/how-can-i-increase-open-files-limit-for-all-processes/8949#8949) solves the issue in this case.
